README.md

```diff
@@ -266,7 +267,43 @@ docker run -p 8080:80 \
    -e OMPI_MCA_btl_vader_single_copy_mechanism=none \
    -e HF_HUB_ENABLE_HF_TRANSFER=1 \
    -e HUGGING_FACE_HUB_TOKEN=$hf_token \
-   -e PREFILL_BATCH_BUCKET_SIZE=1 \
+   --cap-add=sys_nice \
+   --ipc=host \
+   ghcr.io/huggingface/tgi-gaudi:2.0.1 \
+   --model-id $model \
+   --max-input-tokens 4096 \
+   --max-batch-prefill-tokens 16384 \
+   --max-total-tokens 8192
+```
+
+Please implement the inference client according to the [tutorial](https://github.com/huggingface/tgi-gaudi/blob/habana-main/docs/source/basic_tutorials/visual_language_models.md).
+
+### LLava-next 7B FP8 on 1 Gaudi2 card
```

Comment:
Same comment as README. you may list the name as llava-v1.6-mistral-7b in this case